cmake/Reprojucer.cmake

```diff
@@ -1860,12 +1862,6 @@ endfunction()
 
 function(_FRUT_abs_path_based_on_jucer_project_dir in_path out_path)
 
-  if(NOT IS_ABSOLUTE "${in_path}" AND NOT DEFINED JUCER_PROJECT_DIR)
-    message(FATAL_ERROR "The path \"${in_path}\" must be absolute, unless you give "
-      "PROJECT_FILE when calling jucer_project_begin()."
-    )
-  endif()
-
```

Comment:
`Reprojucer.cmake` is meant to be used by developers who are not familiar with CMake and this fatal error is very useful to catch beginner mistakes. We should not delete it. Instead, we should extend it to make it a bit more useful to beginner users, as well as advanced users. See https://github.com/McMartin/FRUT/pull/350.